src/main.ts

```diff
@@ -100,13 +103,31 @@ const executeDockerCommand = async (dockerCommand: string, continueOnError: bool
     var execOptions: any = {
         outStream: new NullOutstreamStringWritable({ decodeStrings: false }),
         listeners: {
-            stdout: (data: any) => console.log(data.toString()), //to log the script output while the script is running.
-            errline: (data: string) => {
+            stdout: async (data: any) => {
+                let scannedResult = {result: null};
+                if(!config.credScanEnable) console.log(data.toString());
+                else if(config.credScanEnable || process.env.CREDSCAN){

```

Comment:
process.env.CREDSCAN -> If the user wants to disable this credscan, he should be able to do it, via setting it false.